### ISSUE TITLE
CONDA_PREFIX not leading to conda env being used

### ISSUE THREAD

**Issue by mbspng** [astral-sh/uv#7829]
The CONDA_PREFIX is not leading to my conda env being used with uv.

I have an activated conda env:

```
-> % echo $CONDA_PREFIX
/home/<user>/miniconda3/envs/ssa
```

I run `uv sync`. I am expecting it to use the conda env. It does not.

```
-> % l -a .venv/bin/python
lrwxrwxrwx 1 mbs 80 okt.   1 09:36 .venv/bin/python -> /home/<user>/.local/share/uv/python/cpython-3.10.13-linux-x86_64-gnu/bin/python3.10*
```

Whereas

```
-> % which python
/home/<user>/miniconda3/envs/ssa/bin/python

-> % l -a ~/miniconda3/envs/ssa/bin/python    
lrwxrwxrwx 1 mbs 10 okt.   1 09:16 /home/<user>/miniconda3/envs/ssa/bin/python -> python3.10*
```

Docs (https://docs.astral.sh/uv/pip/environments/#using-arbitrary-python-environments) say:

> When running a command that mutates an environment such as uv pip sync or uv pip install, uv will search for a virtual environment in the following order:
>
> - An activated virtual environment based on the VIRTUAL_ENV environment variable.
> - An activated Conda environment based on the CONDA_PREFIX environment variable.
> - A virtual environment at .venv in the current directory, or in the nearest parent directory.
>
>  If no virtual environment is found, uv will prompt the user to create one in the current directory via uv venv.

Looks like incorrect behavior to me, judging by the docs. How can I make `uv` use the conda environment?

```
uv version = 0.4.13

Linux <name> 6.8.0-45-generic #45~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Wed Sep 11 15:25:05 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by zanieb** [comment]
`uv sync` intentionally does not respect active environments. See https://docs.astral.sh/uv/concepts/projects/#project-environments for details.

**Comment by mbspng** [comment]
> `uv sync` intentionally does not respect active environments. See https://docs.astral.sh/uv/concepts/projects/#project-environments for details.

Okay, thank you. I think the docs should be updated at the place I linked to, because there it sounds like it. And also after reading the section you linked to, I do not understand it. Seems contradictory to me.

**Comment by zanieb** [comment]
The documentation you linked is in the "pip interface" section, it's all about `uv pip` not the commands outside of it. `uv sync` and `uv pip sync` work very differently.

**Comment by zanieb** [comment]
Yeah sorry it's the downside of supporting both legacy interfaces and new workflows in a single tool. I'll see if I can improve the documentation on that page to make it clear we're just talking about `uv pip`.

**Comment by mbspng** [comment]
I understand. Thanks! Btw. I think it is good that it now supports the poetry / pdm style interface with lock files etc. I evaluated uv a while back before it supported that and decided not to use it at that point, because of the awkward old pip-type-of workflow. Now with the new interface, I re-evaluated.

**Comment by sharifhsn** [comment]
Although `CONDA_PREFIX` is still not supported in the project API, the user workaround with `VIRTUAL_ENV` now works with the `--active` flag as of #11189 (documented at https://docs.astral.sh/uv/concepts/projects/config/#project-environment-path). The pip-project split in the docs makes this behavior more clear as well.

I think this issue is closeable, unless there's interest in making `--active` respect `CONDA_PREFIX` as well.
